addons/event/views/event_event_views.xml

```diff
@@ -200,7 +200,7 @@
                                             <span t-esc="record.address_id.value"/>
                                         </div>
                                     </div>
-                                    <h5 class="o_event_fontsize_11 p-0">
```

Comment:
Is this class still in use somewhere? If not, can you remove it from the scss file?

Reply:
👍 Done